culture/trust_levels.md

```diff
@@ -0,0 +1,108 @@
+the renegades, we’ll receive no short-term reward for any effort spent
+on this. We have *plenty* of other initiatives into which we’d prefer to
+invest our energy.
+
+Therefore, we propose that we change our approach to trust over time,
+and lay out a plan to do so.
+
+# Trust levels defined
+
+We propose three trust levels, which characterize not the technical
+systems built by the SourceCred contributors but rather the composition
+of a community:
+
+  - **Trust level 3** (TL3): Collaborative. It is taken as [common
+    knowledge][common-knowledge] that everyone in the community is
+    acting in good faith. People feel comfortable reflecting on their
+    own incentives: “am I subconsciously skewing my ‘votes’ to benefit
+    myself?”; “how have my motivations for votes, etc. changed over
+    time?”. When asking the same questions of others, we trust that the
+    responses are thoughtfully and truthfully given. Discussions of
+    accidental or subconscious gaming are treated as precious data to be
+    shared and analyzed, never as evidence of any kind of wrongdoing.
+
+  - **Trust level 2** (TL2): Wary. While the community’s core backbone
+    of contributors are still dedicated to the original ideals, perhaps
+    some new members are more interested in expanding their share of the
+    pie rather than the pie as a whole. Primarily distinguished from TL3
+    because good faith is no longer a uniform assumption. Expect skewed
+    voting records, underrecognition of others’ help, quarrels over
+    attribution or allocation.
+
+  - **Trust level 1** (TL1): Alert. This community controls a scarce
+    resource. That resource is considered intrinsically valuable by some
+    people outside the community, who are willing to devote their time
+    to acquiring that resource without much thought for scruples.
+    Primarily distinguished from TL2 because gaming may now be the main
+    goal rather than an opportunistic “side hustle” on legitimate
+    contributions—noting, of course, that legitimate components may yet
+    be a key component of effective gaming strategies! Expect voting
+    cliques, [sybils], chains of obfuscated self-dealing, attempts to
+    corrupt structures of leadership and governance.
+
+[common-knowledge]: https://en.wikipedia.org/wiki/Common_knowledge_(logic)
+[sybils]: https://en.wikipedia.org/wiki/Sybil_attack
+
+(As a point of syntax, note that the numbers decrease as the community
+is understood to be less trustworthy. That is, the number indicates the
+degree of trust, not the passage of time. This is for simplicity that
+“high trust level should mean more trust”, as well as consistency with
+systems like DEFCON or Discourse’s own trust levels.)
+
+# Usage
+
+It’s important to recognize that trust level varies both with time and
+with community. As of late 2019, the SourceCred community is in TL3, but
+(e.g.) some cryptocurrency communities are surely in TL1.
+
```

Comment:
I'd add parens here with context, like "community is in TL3 (high trust)... others in TL1 (low trust)." or something along those lines to still be teaching the reader about what the levels means.